R/dataDrivenClusters.R

```diff
@@ -6,7 +6,7 @@ dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
                                region = NULL) {
 
```

Comment:
Can you write error statements for cases when the user inputs PCA or UMAP < 2? You can use `stop()`

Reply:
got it